Makefile

```diff
@@ -64,80 +124,22 @@ $(GOSEC):
 	$(GO_NOMOD) get -u github.com/securego/gosec/cmd/gosec
 
 .PHONY: sec
-sec: $(GOSEC)
+sec: $(GOSEC) ## Runs gosec to check for potential security issues in the Go source
 	@echo "SECURITY SCANNING"
 	$(GOSEC) -quiet -fmt=csv ./...
 
-.PHONY: clean
-clean:
-	rm -rf bin build release
-
-.PHONY: build
-build: webhooks keeper foghorn tekton-controller gc-jobs
-
-.PHONY: webhooks
-webhooks:
-	$(GO) build -i -ldflags "$(GO_LDFLAGS)" -o bin/$(WEBHOOKS_EXECUTABLE) $(WEBHOOKS_MAIN_SRC_FILE)
-
-.PHONY: keeper
-keeper:
-	$(GO) build -i -ldflags "$(GO_LDFLAGS)" -o bin/$(KEEPER_EXECUTABLE) $(KEEPER_MAIN_SRC_FILE)
-
-.PHONY: foghorn
-foghorn:
-	$(GO) build -i -ldflags "$(GO_LDFLAGS)" -o bin/$(FOGHORN_EXECUTABLE) $(FOGHORN_MAIN_SRC_FILE)
-
-.PHONY: gc-jobs
-gc-jobs:
-	$(GO) build -i -ldflags "$(GO_LDFLAGS)" -o bin/$(GCJOBS_EXECUTABLE) $(GCJOBS_MAIN_SRC_FILE)
-
-.PHONY: tekton-controller
-tekton-controller:
-	$(GO) build -i -ldflags "$(GO_LDFLAGS)" -o bin/$(TEKTONCONTROLLER_EXECUTABLE) $(TEKTONCONTROLLER_MAIN_SRC_FILE)
-
-.PHONY: compile-e2e
-compile-e2e:
```

Comment:
We still need this and `run-e2e-tests` (though the latter also will need to be reworked to have separate targets for running the Tekton tests and the eventual Jenkins tests).

Reply:
Interesting. I thought I search in the whole project for occurrences. Let me bring it back.